### DISCUSSION TITLE
Handling multiple addresses

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#220] [General]
By @kerrilija :

ASA Stats address field can remember multiple addresses. But if I want to add the second address, I can't access "drop down suggestions" from within the app. Therefore, it would be better if we have something like a "+" or "add" button that creates another field which again give us access to the "drop down suggestions" field.

- - - - -
By @marucat27
Maybe one day you can see the various saved addresses in the asa stats app.  Say you have several personal wallets you watch, then a separate nft wallet or wallets, then watch wallets.  This could even be a paid tier?  Thoughts?

**Comment by ipaleka** [comment]
> ASA Stats address field can remember multiple addresses. But if I want to add the second address, I can't access "drop down suggestions" from within the app. Therefore, it would be better if we have something like a "+" or "add" button that creates another field which again give us access to the "drop down suggestions" field.

That drop down suggestions functionality is actually your browser functionality. We don't save any address, actually we don't have any database in this phase of the project and that will probably remain in the pre-DAO phased for free users.
 
> By @marucat27 Maybe one day you can see the various saved addresses in the asa stats app. Say you have several personal wallets you watch, then a separate nft wallet or wallets, then watch wallets. This could even be a paid tier? Thoughts?

Every of our [subscription tiers](https://github.com/asastats/channel/discussions/166) implies having multiple bundles.

From, our very basic [Intro tier description](https://github.com/asastats/channel/wiki/ASA-Stats-Token-utilization#intro):

> - up to three user bundles
> 
> User bundles functionality implies saving bundles in the user settings, as well as editing and changing the addresses in a bundle.

**Reply by marucat27** [reply]
Another suggestion: there could be various tabs for your portfolio that you can categorize.  I could see like an overall portfolio that combined multiple addresses you can add, maybe one for nfts that you want to separate from your main holdings, and one for say watch accounts that do not add value to your holdings.  But these addresses are saved within the app, you would need to add them every time you say launch that app.  You could even save this file or export it?  This could be a separate topic but maybe there could also be a way to create value for the nfts and convert them into a estimated portfolio value of your nfts based on current floor value?

**Reply by ipaleka** [reply]
All of that is covered by our advanced bundle functionality. As of now, our second tier defines up to 8 bundles with up to ten or fifteen addresses in each of them.

**Reply by kerrilija** [reply]
by @ipaleka:

> That drop down suggestions functionality is actually your browser functionality. We don't save any address, actually we don't have any database in this phase of the project and that will probably remain in the pre-DAO phased for free users.
>  

True, but if for every address, there is a seperate form field, suggestions would reappear, and the issue would be solved.

**Reply by ipaleka** [reply]
> > That drop down suggestions functionality is actually your browser functionality. We don't save any address, actually we don't have any database in this phase of the project and that will probably remain in the pre-DAO phased for free users.
> 
> True, but if for every address, there is a seperate form field, suggestions would reappear, and the issue would be solved.

Well, we should discuss whether that is an issue or not. *Making people lazy* by our UI at the expense of our business model (purchasing subscription tiers where they have all of that under their control) is not the way to go if you ask me.

**Reply by marucat27** [reply]
Making customization part of the paid subscription model is good in my opinion.  Many will opt for customizations for a tier.

**Reply by ipaleka** [reply]
Absolutely. our goal is to appreciate pretty much every feasible suggestion from our users for functionalities behind subscription tiers.

At the same time, we should be adding features for free users at a slower pace.
